### DISCUSSION TITLE
.ini kip1= placement.

### DISCUSSION THREAD

**Discussion by ileathan** [hanai3Bi/Switch-OC-Suite#44] [General]
> where to put doesn't matter, but since some users are having problem with the placement, I'm just telling to put at the very end

Maybe it's just something else I did then as the only placement for it to work for me is as I showed. When I placed it under fss0= (maybe under the kip1patch= makes more sense) I would get a black screen.

> Are you sure you put it right below icon=

Yes.

**Comment by fireman644** [comment]
I'm having black screens as well.  Any updates?

**Reply by hanai3Bi** [reply]
update AMS

**Reply by ileathan** [reply]
Idk problem went away for me when I moved the suggested line to the FIRST line of the boot entry but I guess it was not suppose to matter but it did start working when I did that :/

**Comment by hanai3Bi** [comment]
i hadn't checked how exactly hekate parsess thoses ini lines, 
but just put it where it works for you
